html/modules/custom/docstore/src/Controller/DownloadController.php

```diff
@@ -45,33 +101,38 @@ public function __construct(EntityRepositoryInterface $entityRepository,
    *
    * Downloads the latest published version of the file.
    *
-   * @todo throw a 404 Not Found instead?
+   * @param string $media_uuid
+   *   Media uuid.
+   * @param string $provider_uuid
+   *   Provider uuid.
+   * @param string $hash
+   *   Hash.
+   * @param string $filename
+   *   Filename.
+   * @param \Symfony\Component\HttpFoundation\Request $request
+   *   Docstore API request.
+   *
+   * @return \Symfony\Component\HttpFoundation\BinaryFileResponse
+   *   Binary file response.
+   *
+   * @todo consolidate that with the MediaController::getMediaContent()?
+   * @todo $filename is currently ignored.
```

Comment:
Correct, Is only in the URL so it makes sense to users

Reply:
Thanks for confirming, I'll put a note instead of the todo.